cardano-db-sync/src/Cardano/DbSync/Sync.hs

```diff
@@ -209,6 +210,28 @@ dbSyncProtocols syncEnv metricsSetters tc codecConfig version bversion =
     backend = envBackend syncEnv
 
     initAction channel = do
+      consumedFixed <- getIsConsumedFixed syncEnv
+      case consumedFixed of
+        Nothing -> oldActionFixes channel
+        Just wrongEntriesSize | wrongEntriesSize == 0 -> do
+          logInfo tracer "Found no wrong entries"
```

Comment:
might have been nice to have a little more context as to what this is to do with as it might be confusing seeing this message alone in the logs.